bitcash/format.py

```diff
@@ -84,15 +96,18 @@ def wif_to_bytes(wif):
     elif version == TEST_PRIVATE_KEY:
         version = 'test'
     else:
-        raise ValueError('{} does not correspond to a mainnet nor '
-                         'testnet address.'.format(version))
+        raise ValueError('{} does not correspond to a mainnet, testnet, nor '
+                         'regtest address.'.format(version))
 
     # Remove version byte and, if present, compression flag.
     if len(wif) == 52 and private_key[-1] == 1:
         private_key, compressed = private_key[1:-1], True
     else:
         private_key, compressed = private_key[1:], False
 
+    if regtest:
```

Comment:
This doesn't seem right. If we have this flag at all, we should assert if it is no actually a regtest one.

Reply:
I [changed this function around a bit](https://github.com/pybitcash/bitcash/pull/82/commits/e90ce7868f4a091759cb3f8970274c6771a27fec), however WIFs have a preceding 'version' byte that corresponds to either mainnet or testnet ... and unfortunately the **regtest** uses the same 'version' with **testnet**, so I think the WIF functions will be the only remaining function that will require a 'regtest' flag to work properly. [https://github.com/pybitcash/bitcash/pull/82/commits/e90ce7868f4a091759cb3f8970274c6771a27fec](https://github.com/pybitcash/bitcash/pull/82/commits/e90ce7868f4a091759cb3f8970274c6771a27fec)

Reply:
Sounds good with me, thank you.